.github/workflows/ci.yaml

```diff
@@ -170,19 +170,14 @@ jobs:
 
           if [ "${{ matrix.environment }}" = "staging" ]; then
             SANDBOX_ID="STAGING_SANDBOX_ID"
+            CHART_REF="STAGING_CHART_REF"
           fi
           
-          if [ "${{ github.ref }}" = "refs/heads/master"]; then
-            CHART_REF="$GITHUB_SHA"
-          else
-            CHART_REF="$GITHUB_REF"
-          fi
-
-          echo "::set-output name=sandbox-id::$SANDBOX_ID"
-
           yq w -d1 .flux.yaml metadata.name "$DEPLOYMENT_NAME" \
           | yq w -d1 - metadata.namespace "$DEPLOYMENT_NAME-$SANDBOX_ID" \
           | yq w -d0 - metadata.name "$DEPLOYMENT_NAME-$SANDBOX_ID" \
+          | yq w -d0 - metadata.tags.sandboxId "$SANDBOX_ID" \
+          | yq w -d1 - metadata.tags.sandboxId "$SANDBOX_ID" \
```

Comment:
why twice?

Reply:
They're different documents. `d0` is the first one in `.flux.ci` (the namespace), and `d1` is the second (the HelmRelease).